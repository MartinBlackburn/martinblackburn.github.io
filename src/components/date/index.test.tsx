// libraries
import { render } from "@testing-library/react";

// component under test
import DateComponent from "./";

describe("Date component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<DateComponent date={new Date("1986-02-11")} />);

        expect(asFragment()).toMatchSnapshot();
    });
});
