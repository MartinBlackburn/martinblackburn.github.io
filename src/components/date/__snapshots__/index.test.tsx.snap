// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date component renders correctly 1`] = `
<DocumentFragment>
  <time
    datetime="1986-02-11"
  >
    11 February 1986
  </time>
</DocumentFragment>
`;
